mk/util.mk

```diff
@@ -20,6 +20,8 @@ SUPPORTED_PLATFORMS += linux-amd64
 SUPPORTED_PLATFORMS += darwin-386
 SUPPORTED_PLATFORMS += darwin-amd64
 
+SUPPORTED_PLATFORMS += freebsd-amd64
```

Comment:
:+1: I believe we didn't have this before due to some reuseport issues (like usual...).

Reply:
Thanks for commenting. I'm not sure how this relates to reuseport though. This just directs `make test` to see whether it can compile a binary for FreeBSD.

Reply:
IIRC, compiling on FreeBSD was broken at the time these tests were added.